tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -551,6 +563,232 @@ describe(__filename, () => {
     expect(root.find(AddonAuthorLinks)).toHaveProp('addon', addon);
   });
 
+  it('renders related categories', () => {
+    const results = [
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_ANDROID,
+        name: 'Alerts & Update',
+        slug: 'alert-update',
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_ANDROID,
+        name: 'Blogging',
+        slug: 'blogging',
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_ANDROID,
+        name: 'Games',
+        slug: 'Games',
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_FIREFOX,
+        name: 'Alerts & Update',
+        slug: 'alert-update',
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_FIREFOX,
+        name: 'Security',
+        slug: 'security',
+        type: ADDON_TYPE_EXTENSION,
+      },
+      {
+        ...fakeCategory,
+        application: CLIENT_APP_FIREFOX,
+        name: 'Anime',
+        slug: 'anime',
+        type: ADDON_TYPE_STATIC_THEME,
+      },
+    ];
+
+    const addon = createInternalAddonWithLang({
+      ...fakeAddon,
+      categories: {
+        [CLIENT_APP_FIREFOX]: ['alert-update', 'security'],
+      },
+      type: ADDON_TYPE_EXTENSION,
+    });
+
+    dispatchClientMetadata({ clientApp: CLIENT_APP_FIREFOX, store });
+    store.dispatch(loadCategories({ results }));
+
+    const root = render({
+      addon,
+      store,
+    });
+
+    expect(
+      root.find('.AddonMoreInfo-related-categories').find(Link),
+    ).toHaveLength(2);
+    expect(
+      root
+        .find('.AddonMoreInfo-related-categories')
+        .find(Link)
+        .at(0)
+        .prop('to'),
+    ).toEqual('/extensions/category/alert-update/');
+    expect(
+      root
+        .find('.AddonMoreInfo-related-categories')
+        .find(Link)
+        .at(1)
+        .prop('to'),
+    ).toEqual('/extensions/category/security/');
+  });
+
+  it('does not render related categories when add-on has no category', () => {
+    const results = [
```

Comment:
Could you add the new test cases into a `describe` block and define `results` only once? It should look like this:

```js
describe('related categories', () => {
  const results = [
    // ...
  ];

  it('renders related categories', () => {
    // ...
  });

  // other test cases here
});
```

Reply:
You could also rename `results` to `categories` and then dispatch actions as follows:

```js
store.dispatch(loadCategories({ results: categories }));
```